webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const OptimizeJsPlugin = require('optimize-js-plugin');
+const plugins = [new HtmlWebpackPlugin({
+  template: 'src/index.html',
+  filename: 'index.html',
+  inject: 'body'
+})];
+
+module.exports = (env) => {
+  if (env === 'production') {
+    plugins.push(
+      new OptimizeJsPlugin({
+        sourceMap: false
+      })
+    )
+  }
+
+  return {
+    mode: env || 'production',
+    entry: './src/script.js',
+    output: {
+      path: path.resolve(__dirname, 'build'),
+      filename: 'app.bundle.js'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          loader: "babel-loader",
+          options: {
+            plugins: env !== 'production' ? ["react-hot-loader/babel"] : []
+          }
+        },
+        {
+          test: /\.css$/,
+          use: [
+            { loader: 'style-loader' },
+            {
+              loader: 'css-loader',
+              options: {
+                modules: true
+              }
+            }
+          ]
+        }
+      ]
+    },
+    plugins: plugins
```

Comment:
just `plugins`, when you have same name of property and value you can connect it ;)

Reply:
OK